<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileManager6</class>
 <widget class="QWidget" name="ProfileManager6">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gen 6 Profile Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButtonNew">
     <property name="text">
      <string>New</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButtonEdit">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTableView" name="tableView">
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QPushButton" name="pushButtonDone">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonDone</sender>
   <signal>clicked()</signal>
   <receiver>ProfileManager6</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
